config/source/multi/deployments/controller.yaml

```diff
@@ -47,12 +47,16 @@ spec:
 
         # How often (in seconds) the autoscaler tries to scale down the statefulset.
         - name: AUTOSCALER_REFRESH_PERIOD
-          value: '10'
+          value: '100'
 
         # The number of virtual replicas this pod can handle.
         - name: POD_CAPACITY
           value: '100'
 
+        # The scheduling policy type for placing vreplicas on pods (see type SchedulerPolicyType for enum list)
+        - name: SCHEDULER_POLICY_TYPE
+          value: '1'
```

Comment:
I'd like the value to be more descriptive (eg. `maxfillup`). We can do this in another PR.